yb-voyager/cmd/common.go

```diff
@@ -195,20 +195,29 @@ func GetTableRowCount(filePath string) map[string]int64 {
 	return tableRowCountMap
 }
 
-func getExportedRowCountSnapshot(exportDir string) map[string]int64 {
+func getExportedRowCountSnapshot(exportDir string) (map[string]int64, map[string][]string) {
 	tableRowCount := map[string]int64{}
+	leafPartitions := map[string][]string{}
 	for _, fileEntry := range datafile.OpenDescriptor(exportDir).DataFileList {
-		tableRowCount[fileEntry.TableName] += fileEntry.RowCount
+		renamedTable := renameTableIfRequired(fileEntry.TableName)
```

Comment:
nit: it'll be cleaner if `renameTableIfRequired` can return two values (renamedTable, didRenameHappen-bool). Then you can use the second value directly instead of having to compare original vs renamed table in every use of this func